src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAY_RENTAL_COST = 40;
+  const LONG_TERM_DISCOUNT = 50;
+  const LONG_TERM_DURATION = 7;
+  const SHORT_TERM_DURATION = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  if (days >= LONG_TERM_DURATION) {
+    return DAY_RENTAL_COST * days - LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= SHORT_TERM_DURATION) {
+    return DAY_RENTAL_COST * days - SHORT_TERM_DISCOUNT;
```

Comment:
Similar to the long-term discount, ensure that the short-term discount is applied correctly. If the discount is meant to be per day, adjust the calculation.